sheetstorm/management/commands/generate_test_data.py

```diff
@@ -40,6 +40,39 @@ class Command(BaseCommand):
 
     @transaction.atomic
     def handle(self, *args: Any, **options: Union[bool, int, None]) -> None:
+        is_request_to_use_prepared_set = self._get_request_to_create_data_using_prepared_set(options)
+
+        if is_request_to_use_prepared_set:
+            self.create_data_using_prepared_set(options)
+        else:
+            self.execute_creating_users(options)
+            self.execute_creating_project(options)
+
+        logging.info(f"Total number of users in the database: {CustomUser.objects.count()}")
+        logging.info(f"Total number of projects in the database: {Project.objects.count()}")
+
+    def create_data_using_prepared_set(self, options: OptionsDictType) -> None:
+        requested_set = self._get_request_to_create_data_using_prepared_set(options)
+
+        self.execute_creating_users(requested_set)
+        self.execute_creating_project(requested_set)
+
+    @staticmethod
+    def _get_request_to_create_data_using_prepared_set(options: OptionsDictType) -> Any:
+        if options[TestDataSize.SMALL.name] is True:
+            requested_set = SMALL_SET
+        elif options[TestDataSize.MEDIUM.name] is True:
+            requested_set = MEDIUM_SET
+        elif options[TestDataSize.LARGE.name] is True:
+            requested_set = LARGE_SET
+        elif options[TestDataSize.EXTRA_LARGE.name] is True:
+            requested_set = EXTRA_LARGE_SET
```

Comment:
Please create map of this all variables and move them to constants. It will simplify whole method.